.coveragerc

```
@@ -0,0 +1,12 @@
+# .coveragerc to control coverage.py
+[run]
+concurrency=thread,greenlet
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
```

I use `pragma` for Telegram, so this should be
```suggestion
exclude_lines =
    # Have to re-enable the standard pragma
    pragma: no cover
    
```